archetypes/mp-bare/src/main/resources/archetype-resources/src/main/java/Main.java.vm

```diff
@@ -0,0 +1,57 @@
+#set( $symbol_pound = '#' )
+#set( $symbol_dollar = '$' )
+#set( $symbol_escape = '\' )
+package ${package};
+
+import java.io.IOException;
+import java.util.logging.LogManager;
+
+import io.helidon.microprofile.server.Server;
+
+/**
+ * Main method simulating trigger of main method of the server.
```

Comment:
Maybe be more clear here what this class does.  What server?  Why does it need to be simulated?

Reply:
This is from the helidon-mp-quickstart

Reply:
I've update the javadoc comments for the main class of both quickstarts and new archetypes to `The application main class.`